nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -147,16 +151,68 @@ define([
             const appSpec = specs[typesToFiles[selectedFileType].appId];
             const filePathNode = ui.getElement('input-container.file-paths');
             const paramNode = ui.getElement('input-container.params');
+            showConfigMessage();
```

Comment:
Show the message here - it gets rendered every time the widgets get rendered, so we change it with the different file types.